show/main.py

```diff
@@ -56,7 +56,7 @@ def get_routing_stack():
 
 
 # Global Routing-Stack variable
-routing_stack = get_routing_stack()
+routing_stack = "frr"
```

Comment:
need to think about this.

Reply:
this reduce the function, can you exclude this in this pr? also it does not fit in the pr title.

Reply:
removed it. will think of better way of doing this